cdsapi/api.py

```diff
@@ -116,6 +116,14 @@ def check(self):
         self.debug(metadata.headers)
         return metadata
 
+    def update(self):
```

Comment:
Would be nice to be able to pass in a request ID from an external source, not just rely on the instance state. You might want to check back after 24 hours from a different process or so. So instead of (or in addition to) relying on `self.reply`, the user should be able to provide the ID themselves.

Reply:
Should be possible now